src/app/components/mainContent/mainContent.component.tsx

```diff
@@ -16,48 +13,34 @@ export function MainContent(): JSX.Element {
     if (content.pending) {
         return <LoadingIndicator />;
     }
-    if (content?.articles?.length) {
+    if (content.articles.length) {
         return (
             content && (
                 <div className="main-content-container">
-                    <div className="articles-container">
-                        {content?.articles.length >= 3 &&
-                            content.articles
-                                ?.slice(0, 3)
-                                .map((article: ContentItem) => (
-                                    <ContentTile
-                                        contentItem={article}
-                                        contentLabel={ContentLabel.Article}
-                                    />
-                                ))}
+                    <div className="category-items">
```

Comment:
please remove check from line 18 as in this scope we always have content available

Reply:
and please extract nested teplates to functions as we discussed yesterday (if the code seems similar in many places you should create dedicated component)

Reply:
In this situation in main content container are nested only category generic tiles surrounded by div created for the needs of styling. Please onsider this one more time.